include/singa/core/device.h

```diff
@@ -107,12 +109,20 @@ class Device {
   Context* context(int k) { return &ctx_; }
 
   bool graph_enabled() const { return graph_enabled_; }
+  float verbosity() const { return verbosity_; }
```

Comment:
pls add some comments on the verbosity, e.g., what does 0, 1 and 2 mean.

Reply:
yes, I will add

Reply:
added, thanks